resources/views/events/github/branch_protection_rule/edited.blade.php

```diff
@@ -6,19 +6,19 @@
 $repository = $payload->repository;
 ?>
 
-{!! __('tg-notifier::events/github/branch_protection_rule.edited.title', [
-    'repo' => "<a href='$repository->html_url'>$repository->full_name</a>"
+📝 {!! __('tg-notifier::events/github/branch_protection_rule.edited.title', [
+    'repo' => "🦑<a href='$repository->html_url'>$repository->full_name</a>"
         ]
     ) !!}
 
 @if(isset($payload->changes->name->from))
-{!! __('tg-notifier::events/github/issues.edited.changes.title.from', ['title_from' => $payload->changes->name->from]) !!}
-{!! __('tg-notifier::events/github/issues.edited.changes.title.to', ['title_to' => $payload->rule->name]) !!}
+📖 {!! __('tg-notifier::events/github/branch_protection_rule.edited.changes.title.name') !!}
+    📝 {!! __('tg-notifier::events/github/issues.edited.changes.title.from', ['title_from' => $payload->changes->name->from]) !!}
+    🛡 {!! __('tg-notifier::events/github/issues.edited.changes.title.to', ['title_to' => $payload->rule->name]) !!}
 @else
-📢 <b>{{ $payload->rule->name }}</b>
+🛡 {!! __('tg-notifier::events/github/branch_protection_rule.name') !!}: <code>{{ $payload->rule->name }}</code>
 @endif
-
-{!! __('tg-notifier::events/github/branch_protection_rule.edited.link', [
-    'link' => "<a href='$repository->html_url/settings/branch_protection_rules/{$payload->rule->id}'>{$repository->html_url}/settings/branch_protection_rules/{$payload->rule->id}</a>"
+🔗 {!! __('tg-notifier::events/github/branch_protection_rule.edited.link', [
+    'link' => "<a href='$repository->html_url/settings/branch_protection_rules/{$payload->rule->id}'>{$payload->rule->id}</a>"
```

Comment:
We need to add '#' to synchronize with other events

```suggestion
    'link' => "<a href='$repository->html_url/settings/branch_protection_rules/{$payload->rule->id}'>#{$payload->rule->id}</a>"
```